### ISSUE TITLE
About the EPE computing

### ISSUE THREAD

**Issue by jiujing23333** [NVlabs/PWC-Net#25]
As you have mentioned, the EPE is 2.29 and 2.31. Which data set is this performance from? Training set, test set or eval set you customized?

**Comment by deqings** [comment]
The whole training set of MPI-Sintel.
